src/code_80005FD0.c

```diff
@@ -4518,17 +4517,17 @@ void func_800127E0(void) {
     D_80162FCC = 0;
 }
 
-void func_80012A48(TrainCarStuff *trainCar, s16 arg1) {
+void sync_train_car_vehicle_actor(TrainCarStuff *trainCar, s16 orientationY) {
```

Comment:
good name!

Reply:
`sync_rolling_stock_train`?

Reply:
I rename it to `sync_train_car_vehicle_to_actor` to precise it's only one way but rolling_stock_train lose some information like it's only vehicle to actor

Reply:
`sync_rolling_stock_with_train`? to_actor_ isn't great because it's specifically syncing it to the rest of the train?

Reply:
what do you mean the rest ? because it only sync between an vehicleStuff with an actor so the function are call three time one for every car of the train

Reply:
VehicleStuff is probably `Train` as in, the whole train

Reply:
`sync_train_components`